### ISSUE TITLE
Fire a callback when the fuzzy-finder is opened

### ISSUE THREAD

**Issue by ivangalbans** [1egoman/fuzzy-picker#23]
I would like a callback `OnOpen` like `OnClose`.

My question is: How can I execute a function when `fuzzy-finder` is opened ?
This functionality is very useful

**Comment by 1egoman** [comment]
The easiest way is to manage focus on your own - ie, using the "controlled" method. When you handle the keypress or other opening mechanism, you can call whatever initialization logic you'd like.

**Comment by ivangalbans** [comment]
Yes, it works but I think it's easier to have an `OnOpen` callback. If this is possible, of course.

**Comment by 1egoman** [comment]
If it's something that you'd like to do, feel free to make the change and open a pull request - if it looks good I'd merge it! This isn't something that I have a need for though (and don't have time to prioritize it right now).
